dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -454,68 +454,136 @@ public void Test_Incremental_Job_Test_Pages_Are_Found_Create_And_Publish_New_Pag
+                    SiteSearchJobImpl.RUNNING_ONCE_JOB_NAME);
+            jobDataMap.put(SiteSearchJobImpl.INCLUDE_EXCLUDE, "all");
+            jobDataMap
+                    .put(SiteSearchJobImpl.LANG_TO_INDEX, new String[]{Long.toString(defaultLang.getId()),
+                            Long.toString(secondLang.getId())});
+            jobDataMap.put(SiteSearchJobImpl.INDEX_HOST, site.getIdentifier());
 
-            ContainerDataGen.publish(container);
+            final JobDetail jobDetail = Mockito.mock(JobDetail.class);
+            Mockito.when(jobDetail.getJobDataMap()).thenReturn(jobDataMap);
+            final JobExecutionContext context = Mockito.mock(JobExecutionContext.class);
+            Mockito.when(context.getJobDetail()).thenReturn(jobDetail);
+            Mockito.when(context.getFireTime()).thenReturn(new Date());
+            final SiteSearchJobImpl impl = new SiteSearchJobImpl();
+            impl.run(context);
 
-            final String uuid = UUIDGenerator.generateUuid();
+            final List<String> indicesAfterTest = siteSearchAPI.listIndices();
+            Assert.assertFalse(indicesAfterTest.isEmpty());
+            final String newIndexName = indicesAfterTest.get(0);
 
-            final Template template = new TemplateDataGen()
-                    .withContainer(container.getIdentifier(), uuid)
-                    .nextPersisted();
+            SiteSearchResults searchResults = siteSearchAPI.search(newIndexName, "catalina", 0, 10);
+            Assert.assertTrue("Content in default Language gives results", searchResults.getTotalResults() >= 1);
 
-            TemplateDataGen.publish(template);
+            searchResults = siteSearchAPI.search(newIndexName, "catherine", 0, 10);
+            Assert.assertTrue("Content in second Language gives results", searchResults.getTotalResults() >= 1);
 
-            HTMLPageAsset page = new HTMLPageDataGen(folder, template).languageId(lang1.getId())
-                    .nextPersisted();
+        } finally {
+            Config.setProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE",
+                    defaultContentToDefaultLangOriginalValue);
+        }
+    }
+
+    private void createNewVersionAndPublishExistingEmployeeContent(Language secondLang,
+            Contentlet contentletDefaultLang, String firstName) throws DotDataException, DotSecurityException {
+        Contentlet contentletSecondLang = contentletAPI
+                .find(contentletDefaultLang.getInode(), systemUser, false);
+        contentletSecondLang.setInode("");
+        contentletSecondLang.setLanguageId(secondLang.getId());
+        contentletSecondLang.setStringProperty("firstName", firstName);
+        contentletSecondLang = contentletAPI.checkin(contentletSecondLang, systemUser, false);
+        ContentletDataGen.publish(contentletSecondLang);
+    }
+
+    private Contentlet createAndPublishEmployeeContent(Host site, Language language,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'site' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)